lambda_mailer.tf

```diff
@@ -0,0 +1,52 @@
+resource "aws_lambda_function" "sns_to_ses_mailer" {
+  filename = "${var.lambda_sns_to_ses_mailer_zip["base_path"]}/${var.lambda_sns_to_ses_mailer_zip["file_name"]}"
+  function_name = "sns_to_ses_mailer"
+  role = "${aws_iam_role.lambda_sns_to_ses_mailer.arn}"
+  handler = "sns_to_ses_mailer.lambda_handler"
+  runtime = "python2.7"
+  source_code_hash = "${base64sha256(file(format("%s/%s",var.lambda_sns_to_ses_mailer_zip["base_path"],var.lambda_sns_to_ses_mailer_zip["file_name"])))}"
+  publish = true
+  timeout = 300
+  environment {
+    variables = {
+      LOG_LEVEL = "${var.log_level}"
+      REGION = "${var.region}"
+      MAX_THREADS  = "${var.max_threads}"
+    }
+  }
+  depends_on = ["aws_cloudwatch_log_group.sns_to_ses_mailer"]
+}
+
+resource "aws_iam_role" "lambda_sns_to_ses_mailer" {
+  name = "lambda_ucfs_sns_to_ses_mailer"
```

Comment:
Probably don't want 'ucfs' in a generic module's resource name?

Reply:
Good spot, not sure why it ever had that in the name